karapace/protobuf/io.py

```diff
@@ -51,7 +51,7 @@ def find_message_name(schema: ProtobufSchema, indexes: List[int]) -> str:
 def get_protobuf_class_instance(schema: ProtobufSchema, class_name: str, cfg: Dict) -> Any:
     directory = cfg["protobuf_runtime_directory"]
     proto_name = calculate_class_name(str(schema))
-    proto_path = f"{directory}/{proto_name}.proto"
+    proto_path = f"{proto_name}.proto"
     class_path = f"{directory}/{proto_name}_pb2.py"
```

Comment:
what about this directory? it can be absolute?

Reply:
Yes